Portfolio: Work and About Me Sections

devendra-alt opened the pull request:
### [Portfolio: finish mobile version](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md)

- Added Project Work Section
- Used CSS grid layout for the work section (whole layout)
- Used CSS flexbox for the project section (individual)
- Used CSS flexbox in About Section
- Added About ME Section
- Modified README.md

dev-en-m committed:
Added project section

dev-en-m committed:
Added about-me section

dev-en-m committed:
Modified README.md

dev-en-m committed:
Changed typos in README.md

ademibowale commented on index.html:
```diff
@@ -37,15 +37,208 @@
-    <main id="hero">
-      <h1 class="dev-name">Devendra Mulewa</h1>
+    <main id="hero" class="flex popins">
+      <h1 class="title playfair">Devendra Mulewa</h1>
```
- [x] kindly consider using section tag according to the project requirement ,The container of the section is a section tag for  a proper semantic and easy to read.

ademibowale commented on index.html:
```diff
@@ -37,15 +37,208 @@
+        ><img src="assets/images/download.svg" width="24" height="24" alt="" />
+      </button>
+      <div class="details popins" id="langs">
+        <ul>
+          <h5>Languages</h5>
+          <li>JavaScript (ES5/ES6)</li>
+          <li>Ruby</li>
+          <li>HTML</li>
+          <li>CSS</li>
+        </ul>
+      </div>
+      <div class="details popins" id="frameworks">
+        <ul>
+          <h5>Frameworks</h5>
+          <li>React.js</li>
+          <li>Ruby on rails</li>
+          <li>RSpec</li>
+          <li>Capibara</li>
+        </ul>
+      </div>
+      <div class="details popins" id="skills">
+        <ul>
+          <h5>Skills</h5>
+          <li>Database Management</li>
+          <li>CLI</li>
```
- [x]  kindly consider all The technology categories ("Languages", "Frameworks" and "Skills") are in a `ul`, each of them in a `li` according to requirement and this will make you code easy to read

ademibowale commented on index.html:
```diff
@@ -37,15 +37,208 @@
         hesitate to contact me.
       </p>
-      <button id="start-collab">start collabration</button>
+      <button class="g-btn">start collabration</button>
+      <div class="hero-end">
+        <img
+          src="assets/images/hero-end-final.svg"
+          alt="hero section background"
+        />
```
- [x] Kindly fix the image to look exact like the original figma design according to the requirement kindly stick to the design as much as possible this will make your website look professional (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
- [x] your design
![you](https://user-images.githubusercontent.com/92458236/236251213-5f0afa2e-9af8-4ff7-9f6c-a6ac043960b0.jpg)

- [x] figma design
![template 5](https://user-images.githubusercontent.com/92458236/236251342-17044b9d-f635-45fc-a4b2-473734e297d1.jpg)

devendra-alt replied:
@ademibowale I have an issue adding the design image as you suggested, as I am unable to select and download the mobile image from Figma. that's why I have used the desktop version of image which is not optimized for mobile. do you have any suggestions on that? can you help?

ademibowale replied:
Hold on pls

ademibowale replied:
@devendra-alt To download the image from figma is not that hard but I think you need help if you are available now can I call you on zoom to fix this ?but if you have already fix this that cool and great job

devendra-alt replied:
Problem solved! Just making second code review request 😇

ademibowale reviewed (changes requested):
# Required Changes 🔁
# Hello @devendra-alt     😄 

You're doing a great job on your project. We're proud of you. Good job so far! 

There are some issues that you still need to work on to go to the next project but you are almost there!

![just a little bit](https://user-images.githubusercontent.com/92458236/184504866-f1a3e9e9-9072-441d-8ed3-27d483ba70d8.gif)

## Highlights
  ✅ Nice user interface
  ✅Correct use of Github Flow
  ✅ No linter errors
✅ Descriptive Pull Request
  

## Required Changes ♻️




_Check the comments under the review._




## Optional suggestions 
   
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding! 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

dev-en-m committed:
CR requirments

Replaced desktop image with mobile image for hero background, Added ul in about-me details, Replaced main with section

Favourezeugwa reviewed (approved):
## STATUS: APPROVED 🥇 🏆 🍹 

Hi @devendra-alt ,

I am Favour 👩🏽 , your reviewer for this PR.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Yayyy! 💃🏽, Congratulations! 🎉 🎉🎉

![happy](https://user-images.githubusercontent.com/99246359/202491301-53f8c6a2-8ff4-4337-9b6b-9c75c8bdf345.gif)

## Highlights 💯 
- No Linter errors ✔️
- Good commit messages ✔️
- Professional Readme file ✔️
- Descriptive PR ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
